domain/src/test/java/org/oppia/android/domain/question/QuestionAssessmentProgressControllerTest.kt

```diff
@@ -994,6 +1003,515 @@ class QuestionAssessmentProgressControllerTest {
     assertThat(updatedState.ephemeralState.state.interaction.solution.solutionIsRevealed).isTrue()
   }
 
+  @Test
+  fun testRevealedSolution_forWrongAnswer_returnScore2OutOf3() {
+    setUpTestApplicationWithSeed(questionSeed = 0)
+    subscribeToCurrentQuestionToAllowSessionToLoad()
+    startTrainingSession(TEST_SKILL_ID_LIST_2)
+
+    // submit wrong answer for question 2
+    submitNumericInputAnswerAndMoveToNextQuestion(4.0)
+    verify(
+      mockCurrentQuestionLiveDataObserver,
+      atLeastOnce()
+    ).onChanged(currentQuestionResultCaptor.capture())
+    val ephemeralQuestion = currentQuestionResultCaptor.value.getOrThrow()
+    assertThat(ephemeralQuestion.ephemeralState.pendingState.wrongAnswerCount)
+      .isEqualTo(1)
+    val hintAndSolution = ephemeralQuestion.ephemeralState.state.interaction.solution
+    assertThat(hintAndSolution.correctAnswer.correctAnswer)
+      .isEqualTo("3.0")
```

Comment:
Will these fit on the same line? If so, prefer that (in general, prefer to fill up as much line space as possible before line wrapping).

Ditto elsewhere.

Reply:
Fixed